autogpt/agent/agent.py

```diff
@@ -102,19 +102,17 @@ def start_interaction_loop(self):
                 logger.typewriter_log(
                     "NEXT ACTION: ",
                     Fore.CYAN,
-                    f"COMMAND = {Fore.CYAN}{command_name}{Style.RESET_ALL}  "
-                    f"ARGUMENTS = {Fore.CYAN}{arguments}{Style.RESET_ALL}",
-                )
-                print(
-                    "Enter 'y' to authorise command, 'y -N' to run N continuous "
-                    "commands, 'n' to exit program, or enter feedback for "
-                    f"{self.ai_name}...",
-                    flush=True,
+                    f"COMMAND = {command_name} "
+                    f"ARGUMENTS = {arguments}",
                 )
+                # print(
+                #     "Enter 'y' to authorise command, 'y -N' to run N continuous "
+                #     "commands, 'n' to exit program, or enter feedback for "
+                #     f"{self.ai_name}...",
+                #     flush=True,
+                # )
                 while True:
-                    console_input = clean_input(
-                        Fore.MAGENTA + "Input:" + Style.RESET_ALL
-                    )
+                    console_input = "y"
```

Comment:
Same thing here.

Reply:
We can add support for this in .env via continuous mode maybe ?